lib/proposal/engine.rb

```diff
@@ -42,8 +42,13 @@ def proposals
   end
 
   module CanProposeInstanceMethods
-    def proposals
-      Adapter.where proposer_type: self.class.to_s, proposer_id: self.id
+    def proposals received = false
```

Comment:
I thinks agonising over the semantics here to avoid conditionals would be good:

```ruby
def proposals
  Adapter.where proposer_type: self.class.to_s, proposer_id: self.id
end

def propositions
  Adapter.where proposer_type: self.class.to_s, email: self.email
end
```